sass/layout/_headers.scss

```diff
@@ -520,6 +520,12 @@
             .MenuWrapper {
                 flex: 1;
 
+                &.main-menu {
```

Comment:
@nevyangelova I wonder if we shouldn't move these types of styles into a `.scss` file next to the component - i.e. `main_menu.scss` like we're doing with the `widgets` components. This would become `.MainMenu` to match the component name.

Reply:
@Qovaros don't worry about this comment for now - it's just something we're thinking about but haven't made a decision yet 😄